force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +360,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        int index = -1;
         // Loop through the list of words
+        for (int i = 0; i < words.size(); i++) {
+
             // If the current word is the word to find, exit the loop
+            if (words.get(i).equals(wordToFind)) {
+                index = i;
+                break;
+            }
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return index; // Replace null with the variable you used to store the result
```

Comment:
good work.  You can save some code and just return  from the loop when your condition is met. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
  ```